app/packs/src/apps/mydb/elements/details/samples/SampleDetails.js

```diff
@@ -739,59 +748,121 @@ export default class SampleDetails extends React.Component {
     )
   }
 
-  addManualCas(e) {
-    DetailActions.updateMoleculeCas(this.props.sample, e.value);
+  // addManualCas(e) {
+  //   DetailActions.updateMoleculeCas(this.props.sample, e.value);
+  // }
+
+  addHypenToCas(cas, boolean) {
+    const { sample } = this.state;
+    const filterCas = cas.replace(/-/g, '');
+    const firstHypenIndex = filterCas.length - 3;
+    const secondHypenIndex = filterCas.length - 1;
+    console.log(filterCas);
```

Comment:
Some left-over console logging in this method.

Reply:
deleted